app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function ($filter,
+    $q,
+    $uibModal,
+    $rootScope,
+    APIService,
+    DataService,
+    Logger,
+    NotificationsService) {
+
+    var mobileclientVersion = {
+      group: "mobile.k8s.io",
+      version: "v1alpha1",
+      resource: "mobileclients"
+    };
+    var remove = function (apiObject) {
+      if (!$rootScope.AEROGEAR_MOBILE_ENABLED) {
+        Logger("mobile is not enabled");
+        return;
+      }
+      var context = {
+        namespace: _.get(apiObject, 'metadata.namespace')
+      };
+      NotificationsService.hideNotification("remove-mobileclient-error");
+      DataService.delete(mobileclientVersion, apiObject.metadata.name, context, {
+        gracePeriodSeconds: 0
+      }).then(
+        // success
+        _.noop,
```

Comment:
the resource is removed immediately as it has no need for a grace period, so I think it disappearing is enough rather than showing an alert / notification too

Reply:
We usually notify. There's some debate about whether that's the right thing, but I'd notify to be consistent.